guides/common/modules/con_content-sources-in-project.adoc

```diff
@@ -0,0 +1,14 @@
+[id="content-sources-in-{ProjectNameID}_{context}"]
+= Content sources in {Project}
```

Comment:
Thoughts on renaming this "Sources of content" rather than "Content sources"?

There are other things which we call "content sources" -- Smart Proxy servers from which a host gets content; Alternate Content Sources -- which have nothing to do with this. So I think we could eliminate at least a little bit of inconsistency and confusion here.

Reply:
Perhaps we could rename it to "Products and repositories in {ProjectName}" and re-use it in _Managing content_ > _Importing content_ instead of the current module that is in there?